cluster/manifests/skipper-canary-controller/canary-cronjob.yaml

```diff
@@ -25,7 +26,7 @@ spec:
           containers:
           - name: skipper-canary-controller
             terminationMessagePolicy: FallbackToLogsOnError
-            image: container-registry.zalando.net/gwproxy/skipper-canary-controller:main-20
+            image: "{{ $image }}"
```

Comment:
Automatic updates for plain images work already, see e.g. https://github.com/zalando-incubator/kubernetes-on-aws/pull/8161

In this PR https://github.com/zalando-incubator/kubernetes-on-aws/pull/8151 I used `$image` variable to extract version out of it.

For this PR we can also add version labels to make it consistent with other manifests.

Reply:
I didn't see the other PR, then I will leave it as it is for now, since we use it in one place